src/remix/codemods/handle-error.ts

```diff
@@ -17,6 +17,8 @@ import chalk from 'chalk';
 import { generateCode } from 'magicast';
 
 export function instrumentHandleError(
+  // MagicAst returns `ProxifiedModule<any>` so therefore we have to use `any` here
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   originalEntryServerMod: ProxifiedModule<any>,
```

Comment:
This isn't great and actually a problem in many wizard flows. I played around with somehow typing this better but didn't get anywhere yet. While magicast is nice to use for small changes, it's quite painful with typing and its abstractions on top of recase. I think we should consider ditching it in favour of recast so I don't want to spend too much time on fixing these types. If anyone has ideas how to type the proxified modules properly, I'm all ears :D